<launch>
 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="burger_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>
  <node pkg="burger_joy" type="burger_teleop_joy" name="teleop"/>

 <!-- turtle_teleop_key -->
  <node pkg="turtlesim" type="turtle_teleop_key" name="turtle_teleop_key"/>

</launch>